### PR TITLE
Fix/monitoring&logging

### PR DESCRIPTION
**Pull request by Cellebyte**
* Added better log messages to VRFRouteConfiguration and Layer2NetworkConfiguration
* Try to fix frr-exporter not correctly terminating sometimes
* Reduced exported routes from netlink neighbors endpoint

### CODE REVIEW

File: cmd/frr-exporter/main.go [hunk 1]
```diff
@@ -63,6 +63,6 @@ func main() {
 	err = server.ListenAndServe()
 	// Run server
 	if err != nil {
-		log.Fatal(fmt.Errorf("Can't start server %w", err))
+		log.Fatal(fmt.Errorf("failed to start server %w", err))
```
Comment (schrej) [inline cmd/frr-exporter/main.go:66]: usually you would put a colon before the wrapped error
```suggestion
		log.Fatal(fmt.Errorf("failed to start server: %w", err))
```

**Review by schrej — APPROVED** [review]

**Comment by Cellebyte** [comment]
Looks like frr-exporter netlink neighbors are now broken

**Comment by Cellebyte** [comment]
> Looks like frr-exporter netlink neighbors are now broken

Still missing ^^

**Review by chdxD1 — APPROVED** [review]
lgtm

**Pull request merged**